debezium-core/src/main/java/io/debezium/relational/RelationalDatabaseConnectorConfig.java

```diff
@@ -549,12 +551,14 @@ public static DecimalHandlingMode parse(String value, String defaultValue) {
             .create();
 
     private final RelationalTableFilters tableFilters;
+    private final ColumnNameFilter columnFilter;
     private final TemporalPrecisionMode temporalPrecisionMode;
     private final KeyMapper keyMapper;
     private final TableIdToStringMapper tableIdMapper;
 
     protected RelationalDatabaseConnectorConfig(Configuration config, String logicalName, TableFilter systemTablesFilter,
-                                                TableIdToStringMapper tableIdMapper, int defaultSnapshotFetchSize) {
+                                                TableIdToStringMapper tableIdMapper, int defaultSnapshotFetchSize,
+                                                ColumnFilterMode columnFilterMode) {
```

Comment:
I feel we are proliferating consturctor params too much. Can't the connector expose a method like `columnFilterMode()` so it wouldn't be necessary to pass the param? Also `SCHEMA` will be the default one and `CATALOG` will be override for two connectors.

Reply:
Hum, hum. I'm not a fan of the method approach, as it's another method exposed on the API surface of `RelationalDatabaseConnectorConfig` and could be called by other classes (which it shouldn't). How about I add another constructor without the parameter, which does the defaulting to `SCHEMA`?